Makefile

```diff
@@ -1075,6 +1089,11 @@ ifeq ($(USE_FREETYPE),1)
   RENDERER_LIBS += $(FREETYPE_LIBS)
 endif
 
+ifeq ($(USE_AUTOUPDATER),1)
```

Comment:
There are two different things in the Makefile: BUILD_* (which means "build this binary...dedicated server, client, QVM compiler, DLLs, etc), and USE_* (which means "turn on this feature in ioq3...VoIP, Ogg Vorbis, etc). I tried to split the autoupdater the same way, since it has a small piece of code in ioquake3 _and_ a separate binary.

As such...I think this is right as-is? I don't feel strongly about it if consensus is this should be changed, though!